utils/src/test/scala/za/co/absa/enceladus/utils/types/FormatSuite.scala

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright 2018-${today.year} ABSA Group Limited
```

Comment:
The year did not "translate". it is `$today.yer`, without `{}` in IntelliJ IDEA. There are more occurrences.

Reply:
Thanks for noticing. It's packed for me upon new file creation, never looked there.